server/src/main/java/tw/waterball/survive/io/server/Client.java

```diff
@@ -1,17 +1,21 @@
 package tw.waterball.survive.io.server;
 
-import tw.waterball.survive.io.domain.model.gameobject.GameObject;
-import tw.waterball.survive.io.domain.model.gameobject.role.Player;
-import tw.waterball.survive.io.domain.model.gameobject.role.Role;
-import tw.waterball.survive.io.domain.model.gameobject.weapon.Weapon;
+import tw.waterball.survive.io.server.worldobject.WorldObject;
+import tw.waterball.survive.io.server.worldobject.role.Player;
+import tw.waterball.survive.io.server.worldobject.role.Role;
+import tw.waterball.survive.io.server.worldobject.weapon.Weapon;
 
 public interface Client {
 
     void onLoginSuccess();
 
     void onHeartBeat();
 
-    void onSufferJury(Role role, int damage);
+    void onMoveForwardOrBackward(Role role);
```

Comment:
I don't get it, 
how does client know the direction the player moves? (backward or forward? )